src/components/radio-group-item/radio-group-item.tsx

```diff
@@ -84,10 +123,12 @@ export class RadioGroupItem {
             "label--outline": appearance === "outline"
           }}
         >
-          {this.icon && this.iconPosition === "start" ? iconEl : null}
+          {iconAtStart}
```

Comment:
@anveshmekala could there possibly be `{iconAtStart}` and `{this.iconStart ? iconStartEl : null}` both rendered?

I was thinking only one of them would be rendered right? If so, can we move the iconStartEl to the `iconAtStart` const above? Same with the end icons.

To me, it just seems confusing with iconStartEl and iconAtStart. It seems like only one of them should only be possible at a time.

Reply:
Yup, there is a possibility of both `iconAtStart `and `iconStartEl `being rendered because we are not maintaining a single source of truth for the `radio-group-item` component due to the complexity. Added a note next to the` deprecated` tag advising users to not use both of them but one or the other.

Reply:
ok. makes sense then. 👍